pyccel/commands/argparse_helpers.py

```diff
@@ -0,0 +1,177 @@
+    """
+    def check_path(path_str):
+        """
+        Check if path_str describes the path to an existing file with the expected suffix.
+        """
+        path = pathlib.Path(path_str)
+        if not path.is_file():
+            raise argparse.ArgumentTypeError(f"File not found: {path}")
+        if path.suffix not in suffixes:
+            raise argparse.ArgumentTypeError(f"Wrong file extension for file: {path}. Expecting one of: {', '.join(suffixes)}")
+        return path.absolute()
+    return check_path
+
+# -----------------------------------------------------------------------------------------
+def add_version_flag(parser):
+    """
+    Add version flag to argument parser.
+
+    Add version flag to argument parser.
+
+    Parameters
+    ----------
+    parser : argparse.ArgumentParser
+        The parser to be modified.
+    """
+    version = pyccel_version
+    libpath = pyccel_path[0]
+    python  = f'python {sys.version_info.major}.{sys.version_info.minor}'
+    message = f'pyccel {version} from {libpath} ({python})'
+
+    parser.add_argument('-V', '--version', action='version', help='Show version and exit.', version=message)
+
+# -----------------------------------------------------------------------------------------
+def add_compiler_selection(parser):
+    """
+    Add compiler selection flags to argument parser.
+
+    Add flags to argument parser to select a compiler. This can be done by family
+    or with a json config file.
+
+    Parameters
+    ----------
+    parser : argparse.ArgumentParser
+        The parser to be modified.
+    """
+    default_compiler = os.environ.get('PYCCEL_DEFAULT_COMPILER', 'GNU')
+    json_file_checker = check_file_type(('.json',))
+
+    group = parser.add_argument_group('Compiler configuration (mutually exclusive options)')
+    compiler_group = group.add_mutually_exclusive_group(required=False)
+    compiler_group.add_argument('--compiler-family',
+                                dest='compiler_family',
+                                choices=available_compilers.keys(),
+                                type=str,
+                                default=default_compiler,
+                                help=f'Compiler family (default: {default_compiler}).')
+    compiler_group.add_argument('--compiler-config',
+                                dest='compiler_family',
+                                type=lambda p: str(json_file_checker(p)),
+                                default=None,
+                                metavar='CONFIG.json',
+                                help='Load all compiler information from a JSON file with the given path (relative or absolute).')
+
+# -----------------------------------------------------------------------------------------
+def add_accelerator_selection(parser):
+    """
+    Add accelerator flags to argument parser.
+
+    Add flags to argument parser to select any accelerators to be used in compilation.
```

Comment:
Here, too, I would emphasize the fact that an argument group is added:

```suggestion
    Add group of accelerator flags to argument parser.

    Add argument group to parser, with flags for selecting any accelerators to be used
    in compilation.
```